backend/services/syson-services/src/main/java/org/eclipse/syson/services/ToolService.java

```diff
@@ -401,4 +401,24 @@ private Stream<EObject> eAllContentStreamWithSelf(EObject o) {
         }
         return Stream.concat(Stream.of(o), StreamSupport.stream(Spliterators.spliteratorUnknownSize(o.eAllContents(), Spliterator.NONNULL), false));
     }
+
+    protected boolean isCompliantAsChild(Element element, Node parentNode, DiagramDescription diagramDescription,
```

Comment:
Could you move them with other protected methods upper in the file please?
Could you comment the method please?

Reply:
Done